climate-data-ca/resources/js/child-functions.js

```diff
@@ -680,9 +680,23 @@ function displayChartData(data, varDetails, download_url, query, container) {
         }
     });
 
-    // 30y averages is by default, so trigger the change event on load
-    $('#chartoption2-' + query['var']).trigger('change');
-
+    if (
+        $('body').find('#toggle-b').length &&
```

Comment:
While it works... this selector is not very descriptive for the reader.
Can we use  $('input[name="absolute_delta_switch"]:checked').val()  instead, since it's this one used elsewhere in the code?

Reply:
I will take care of this one.
I will make sure this variable is part of "query"